TestAssets/TestProjects/AllResourcesInSatellite/Strings.en.resx

```diff
@@ -1,5 +1,103 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
+  <!-- 
```

Comment:
What are the changes to this file?  Are the just the result of regenerating the file with the tooling?